cypress/e2e/security_pages_spec.cy.ts

```diff
@@ -0,0 +1,12 @@
+import { PATHS } from '../../src/app/_constants/paths'
+import { testPageMetadata } from '../support/test-utils'
+
+describe('Security - Main Page', () => {
+  testPageMetadata({ path: PATHS.SECURITY, overrideDefaultTitle: true })
+})
+
+describe('Security - Coordinated Disclosure Policy Page', () => {
+  testPageMetadata({
+    path: PATHS.COORDINATED_DISCLOSURE_POLICY,
+  })
```

Comment:
It looks like it's missing `overrideDefaultTitle: true` in the test?

It's used in the `createMetadata` for this page.

```suggestion
  testPageMetadata({
    path: PATHS.COORDINATED_DISCLOSURE_POLICY,
    overrideDefaultTitle: true,
  })
```

Reply:
Yes! That's the reason. Thank you.